src/layup_cmdline/predict.py

```diff
@@ -175,6 +175,16 @@ def main():
         required=False,
     )
 
+    optional.add_argument(
+        "-us",
+        "--units-sexagesimal",
+        help="Option for output units of RA and Dec to be given in sexagesimal. False will use degrees.",
+        dest="units",
```

Comment:
because you have dest units - it's looking for the user to put some after the -us on the command line. There isn't anything for the user to specify. You want this to be like force flag. The reason why is it it defaults to false I don't need to raise the flag unless I want it to be true